### ISSUE TITLE
Task - Footer - Date Modified text lacks sufficient contrast ratio

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/TalentCloud#5425]
# Description
_Date Modified_ text in footer does not have a sufficient colour contrast ratio.

# Images
![Screen Shot 2021-03-12 at 11 28 54](https://user-images.githubusercontent.com/3046459/110969110-4edc7a00-8326-11eb-9056-87ecc83de38c.png)

**Comment by mnigh** [comment]
Moved this issue into large [colour contrast issue](https://github.com/GCTC-NTGC/TalentCloud/issues/5418).
